<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAuSectionRdfEditWidget</class>
 <widget class="QWidget" name="CAuSectionRdfEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>281</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="descrLabel">
       <property name="text">
        <string>Section description:</string>
       </property>
       <property name="buddy">
        <cstring>descrRichTextWidget</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="undoToolButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="redoToolButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="boldToolButton">
         <property name="text">
          <string>Bold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="italicToolButton">
         <property name="text">
          <string>Italic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="underlineToolButton">
         <property name="text">
          <string>Underline</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="leftAlignToolButton">
         <property name="text">
          <string>LeftAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="centerAlignToolButton">
         <property name="text">
          <string>CenterAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="rightAlignToolButton">
         <property name="text">
          <string>TightAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KRichTextWidget" name="descrRichTextWidget">
       <property name="clickMessage">
        <string>Enter here your plans</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="buddy">
        <cstring>statusComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusComboBox"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="actorsLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Actors:</string>
         </property>
         <property name="buddy">
          <cstring>actorsListWidget</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="actorsListWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>KTextEdit</extends>
   <header>krichtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRichTextWidget</class>
   <extends>KRichTextEdit</extends>
   <header>krichtextwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
